controllers/grid/settings/user/UserGridHandler.php

```diff
@@ -140,6 +144,56 @@ public function initialize($request, $args = null)
             )
         );
 
+        // Roles.
+        $columnBasedGridCellProvider = new ColumnBasedGridCellProvider();
+        $this->addColumn(
+            new class (
+                'roles',
+                'user.roles',
+                null,
+                null,
+                $columnBasedGridCellProvider,
+                contextId: $request->getContext()->getId()
+            ) extends GridColumn {
+                /** @var int|null contextId */
+                private ?int $_contextId;
+
+                /**
+                 * Constructor
+                 *
+                 * @param string $id Grid column identifier
+                 * @param string $title Locale key for grid column title
+                 * @param string $titleTranslated Optional translated grid title
+                 * @param string $template Optional template filename for grid column, including path
+                 * @param GridCellProvider $cellProvider Optional grid cell provider for this column
+                 * @param array $flags Optional set of flags for this grid column
+                 * @param int|null $contextId Optional context identifier for this grid column
+                 */
+                public function __construct(
+                    $id = '',
+                    $title = null,
+                    $titleTranslated = null,
+                    $template = null,
+                    $cellProvider = null,
+                    $flags = [],
+                    int $contextId = null
+                ) {
+                    $this->_contextId = $contextId;
+
+                    parent::__construct($id, $title, $titleTranslated, $template, $cellProvider, $flags);
+                }
```

Comment:
Given that we're using a lot of `Application::get()->getRequest()` in the codebase, I think it's ok to use it on [the line 190](https://github.com/pkp/pkp-lib/pull/9385/files#diff-9bdf4df1fd7942644c46d93df32286ddd88a6f6f5fd6ad7dacf3b9c378b4b8a2R190) to get the `Request > Context` and remove all these lines that I highlighted. Then I don't have nothing more to comment 😁

Reply:
Hello, I'm glad we can remove all those lines!